src/java/htsjdk/variant/variantcontext/VariantContext.java

```diff
@@ -721,33 +721,125 @@ public String getID() {
     // get routines to access context info fields
     //
     // ---------------------------------------------------------------------------------------------------------
+    
+    /** shortcut to {@link CommonInfo#getName()} 
+     * @return the source name
+     */
     public String getSource()                   { return commonInfo.getName(); }
+    
+    /** shortcut to {@link CommonInfo#getFiltersMaybeNull()} 
+     *  @return a <b>modifiable</b> Set of filters. Can be  null. All changes in this set will be reflected in the CommonInfo 
+     */
     public Set<String> getFiltersMaybeNull()    { return commonInfo.getFiltersMaybeNull(); }
+    
+    /** shortcut to {@link CommonInfo#getFilters()} 
+     * @return an unmodifiable Set of filters. Can be empty by never null
+     */
     public Set<String> getFilters()             { return commonInfo.getFilters(); }
+    
+    /** shortcut to {@link CommonInfo#isFiltered()}
+     * @return true if any filter been defined  <code>(!filters.isEmpty())
```

Comment:
"true if filters were applied and the VC did not pass filters"

Fix broken `<code>` tag